.github/workflows/pr_ete_test.yml

```diff
@@ -58,6 +58,7 @@ jobs:
           python3 -m pip install -r requirements/build.txt
           mkdir build
           cd build
+          cp -r /nvme/qa_test_models/offline_pkg/_deps .
```

Comment:
What's the motivation for?

Reply:
> What's the motivation for?

the network is not stable, Cache cutlass repo and replace before build. Maybe need maintain the repo manually in the future.

Reply:
Make a copy of cutlass in gitlab?

Reply:
> Make a copy of cutlass in gitlab?

no, save build/_deps folder when the network is fine. The build script will reuse the _deps folder instead of git clone.